packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/models/ClassInfoModel.java

```diff
@@ -431,7 +441,7 @@ public String toString() {
 
     protected abstract List<MethodInfoModel> prepareMethods();
 
-    protected abstract PackageInfoModel preparePackage();
+    protected abstract List<PackageInfoModel> prepareAncestors();
```

Comment:
Same here. Also, I usually try to maintain same naming for `get` and `prepare` methods.
```suggestion
    protected abstract List<PackageInfoModel> prepareAllAvailablePackages();
```

Reply:
Ok I'll sync them once we choose the right one for the public method.